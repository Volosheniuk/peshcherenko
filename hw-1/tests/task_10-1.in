q 23 2.1 2.34
