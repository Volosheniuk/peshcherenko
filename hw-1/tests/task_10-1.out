q
23
2.100000
2.340000
